plugins/user_mapping/src/user_claim.cpp

```diff
@@ -6,27 +6,53 @@
 #include <optional>
 #include <string>
 
+#include <boost/regex.hpp>
+
```

Comment:
You can remove this line since it's just one additional boost header.

Reply:
Done.